src/enclave/enclave.h

```diff
@@ -221,7 +221,7 @@ namespace enclave
 
         if (start_type == StartType::Join)
         {
-          node.join({ccf_config});
+          node.join({consensus_type, ccf_config});
```

Comment:
Would be better to get that from the join RPC.

Reply:
https://github.com/microsoft/CCF/issues/925